rust-keybroker/keybroker-server/src/main.rs

```diff
@@ -99,6 +99,14 @@ async fn submit_evidence(
 
     let evidence_bytes = URL_SAFE_NO_PAD.decode(evidence_base64).unwrap(); // TODO: Error handling needed here in case of faulty base64 input
 
+    // Optionally dump the evidence to file.
+    // This can be useful for debugging or for educational purpose for example.
+    if data.args.dump_evidence_cbor {
+        let filename = format!("evidence-{challenge_id}.cbor",);
```

Comment:
leftover comma?

Reply:
Well spotted --- clippy or the compiler did not flag it ! I'll fix it